src/mlpack/bindings/cli/map_parameter_name.hpp

```diff
@@ -53,14 +53,14 @@ std::string MapParameterName(
 }
 
 /**
- * Map the parameter name to the name seen by boost::program_options.
- *
+ * Map the parameter name to the name seen by CLI.
+ *  
```

Comment:
```suggestion
 *
```

Remove some extra whitespace.